www/codeigniter/application/controllers/master/Account.php

```diff
@@ -3,48 +3,173 @@
 
 class Account extends MY_Controller 
 {	
+
 	public function __construct()
 	{
 		parent::__construct();
+		$this->force_admin();
 	}
 
 	public function index()
 	{
+		$keyword = null;
+		if($this->input->get('keyword'))
+		{
+			$keyword =  $this->input->get('keyword');
+		}
+		$count_by_keyword = $this->accounts_model->count_by_keyword($keyword);
+		$this->load_pagination('/master/account?keyword='.$keyword,$count_by_keyword);
+		$start = $this->pagination->get_last_page_start($this->input->get('start'),$count_by_keyword);
```

Comment:
@trandinhlam 
lấy $start từ $this->input
trường hợp $start > $this->pagination->get_last_page_start() thì thay đổi $start